ports/geckolib/glue.rs

```diff
@@ -2216,14 +2214,14 @@ fn fill_in_missing_keyframe_values(all_properties:  &[TransitionProperty],
         return;
     }
 
-    let keyframe = match offset {
-        0. => unsafe {
+    let keyframe = if is_initial {
```

Comment:
Why not keeping the `offset` parameter and just do `if offset == 0.` here? I think boolean arguments hard to read.